src/plugin.js

```diff
@@ -199,6 +201,17 @@ class PlaylistMenuItem extends Component {
     titleEl.setAttribute('title', titleText);
     titleContainerEl.appendChild(titleEl);
 
+    // We add thumbnail video description only if specified in playlist options
+    if (showDescription) {
+      const descriptionEl = document.createElement('div');
+      const descriptionText = item.description || '';
+
+      descriptionEl.className = 'vjs-playlist-description-visible';
```

Comment:
I'm wondering if we need to call this `vjs-playlist-description-visible`. I see that we have some css for `vjs-playlist-description` already, but I don't know if that gets used at all? Perhaps we can remove that and just call this `vjs-playlist-description`

Reply:
I don't think it is used, let me check that.